RocketChat/Rocket.Chat
# [FIX] Notifications are not being filtered

## Proposed changes (including videos or screenshots)


 - Add a migration to update the `Accounts_Default_User_Preferences_pushNotifications` setting's value to the `Accounts_Default_User_Preferences_mobileNotifications` setting's value;
 - Remove the `Accounts_Default_User_Preferences_mobileNotifications` setting (replaced by `Accounts_Default_User_Preferences_pushNotifications`);
 - Rename 'mobileNotifications' user's preference to 'pushNotifications'.



## Issue(s)

Fix #23447 
Fix #23509
Introduced in #22978 
[Task - ClickUp](https://app.clickup.com/t/1q2jbpp)

## Steps to test or reproduce

1. Before accessing this branch, check the **Administration > Accounts > Default User Preference > Mobile Notifications Default Alert** and **Push Notifications Default Alert** settings' value;
2. Access this branch and restart the server;
3. The "Mobile Notifications Default Alert" setting should be erased and its previous value should be inserted into the "Push Notifications Default Alert" setting.

## Further comments

A notifications cap reset for `4.0.x` workspaces should also be performed along with this fix.
